app.js

```diff
@@ -14,7 +14,7 @@ app.get('/api/topics', getAllTopics);
 
 app.get('/api', getEndpoints);
 
-app.get('/api/articles/:article_id', getArticleById);
+app.get('/api/articles/:article_id', getAllArticles);
```

Comment:
We are still looking for a single article here, keeping the single article and all article functions separate.